pkgs/by-name/ne/nerdctl/package.nix

```
@@ -6,38 +6,45 @@
   installShellFiles,
   buildkit,
   cni-plugins,
+  writableTmpDirAsHomeHook,
+  versionCheckHook,
   extraPackages ? [ ],
 }:
 
-buildGoModule rec {
+buildGoModule (finalAttrs: {
   pname = "nerdctl";
-  version = "1.7.7";
+  version = "2.2.0";
 
   src = fetchFromGitHub {
     owner = "containerd";
     repo = "nerdctl";
-    rev = "v${version}";
-    hash = "sha256-GHFs8QvLcXu+DZ851TCLI7EVc9wMS5fRC4TYBXzyv3Q=";
+    rev = "v${finalAttrs.version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```
using this avoids issues where there's a tag and branch with the same name

(please double check the hash doesn't change)